models/classes/cat/GetDeliveryExecutionsItems.php

```diff
@@ -74,17 +74,25 @@ public function __construct(
      */
     public function getItemsRefs()
     {
-        $sections = $this->assessmentTestSession->getAssessmentTest()->getComponentsByClassName('assessmentSection');
-        $items = [];
-        foreach ($sections as $section) {
-            $assessmentItemsRef = $this->getSectionItems($section, $this->deliveryExecution->getDelivery());
```

Comment:
It may be out of topic, but can you please remote the method `getSectionItems` because it's not used anymore.